src/blackberry10/index.js

```diff
@@ -57,6 +57,7 @@ module.exports = {
             modelName = getModelName(),
             uuid = getUUID(),
             info = {
+                manufacturer: 'Research in Motion',
```

Comment:
For reference, `Research in Motion` is not the name of the company anymore, it's `BlackBerry`:

http://press.blackberry.com/press/2013/research-in-motion-changes-its-name-to-blackberry.html